pkg/operator/sync.go

```diff
@@ -392,6 +392,18 @@ func newPodTemplateSpec(config *OperatorConfig, features map[string]bool) *corev
 				},
 			},
 		},
+		{
+			Name: "trusted-ca",
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					Items: []corev1.KeyToPath{{Key: "ca-bundle.crt", Path: "tls-ca-bundle.pem"}},
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "machine-api-operator-trusted-ca",
```

Comment:
`machine-api-operator-trusted-ca` is also used here https://github.com/openshift/machine-api-operator/pull/646/commits/145fae715a77084768d974a1a9c8124aa62b950a#diff-fa45321336db7ad1cedc28bf643a4f97R109. Can we put it in a constant?